app/audit/AuditingService.scala

```diff
@@ -70,6 +73,19 @@ class AuditingService @Inject()(appConfig: FrontendAppConfig, auditConnector: Au
     }
   }
 
+  private def eventCheckLength(extendedDataEvent: ExtendedDataEvent): Unit = {
+    if (extendedDataEvent.toString.length > 32665) {
```

Comment:
might be a good idea to move this "magic number" into app config ?
so, this would look like:

`if (extendedDataEvent.toString.length > appConfig.maxAuditEventSizeBytes)`